test/e2e/config/conf.js

```diff
@@ -86,7 +86,7 @@ const config = {
     strict: true,
     // format: ['node_modules/cucumber-pretty'],
     format: ['node_modules/cucumber-pretty', 'json:reports_json/results.json'],
-    tags: ['@all'],
+    tags: ['@ignore'],
```

Comment:
@udaydenduluri33  please add @all, we want to run tests on build.

Reply:
Done